rabbitmq/check.py

```diff
@@ -119,6 +119,14 @@ def _get_config(self, instance):
         password = instance.get('rabbitmq_pass', 'guest')
         custom_tags = instance.get('tags', [])
         parsed_url = urlparse.urlparse(base_url)
+        if not parsed_url.scheme:
+            self.log.warning('The rabbit url did not include a protocol, assuming http')
+            # urlparse.urljoin cannot add a protocol to the rest of the url for some reason.
+            # This still leaves the potential for errors, but such urls would never have been valid, either
+            # and it's not likely to be useful to attempt to catch all possible mistakes people could make
+            base_url = 'http://' + base_url
```

Comment:
How does this play with kubernetes/autoconfig? See https://github.com/DataDog/integrations-core/issues/838